genai-on-vertex-ai/developer_productivity_with_genai/README.md

```diff
@@ -1,75 +1,22 @@
-# developer-productivity-with-genai
-Those sample assets can be used to show builders and partners how to solve different developer tasks such as code generation, code explanation, unit test generation, comment generation, code debugging, code migration and talk to code and doc in software development life cycles to increase developer productivity with Codey APIs and other GCP services.
+# Developer Productivity with GenAI
 
-## Contents
-### E2E codey api notebook covers
-All User Journeys mentioned below such as code generation, fine-tuning, debugging, migration, and talk to code and doc.
+Codey models are text-to-code models from Google AI, trained on a massive code related dataset. You can generate code related responses for different scenarios such as writing functions, debugging, explaining code etc. Here is [the overview](https://cloud.google.com/vertex-ai/docs/generative-ai/code/code-models-overview) of all the Codey APIs.
 
-### wireframe to live website notebook covers 
-User Journey - Code Generation support for more languages (HTML , CSS, Objective-C etc):
+We offer a comprehensive set of notebooks that demonstrate how to use Codey APIs to solve various tasks in software development life cycle including code generation, unit test generation, code explanation, comment generation, code refactoring, debugging, migration, codebase chat, Doc search, and JIRA search.
 
-Tech Tasks:
-- Generate functions
-- Explain code
-- Generate unit tests
-- Assisted code refactoring and modification
+Notebooks:
```

Comment:
Please add links to the notebooks for better navigation.

Reply:
Done. Please check.